Classes/Utility/ImageVariantsUtility.php

```diff
@@ -20,7 +20,8 @@ class ImageVariantsUtility
      */
     protected static $allowedVariantProperties = [
         'breakpoint',
-        'width'
+        'width',
+        'widthRetina'
```

Comment:
I am still unsure about this property, Do you think it really makes sense to set a with here?
What about just doing something like:

### Reducing configuration by just enabling different variations.

```
medium {
    breakpoint = 768
    width = 680
    highres = 1
    superhighres = 1
}
```

### Configuring variants with multiplier for more control about the image sizes

```
medium {
    breakpoint = 768
    width = 680
    variants {
       1x = 1
       2x = 1.5
       3x = 2
    }
}
```

Reply:
Great idea! I'll change my code and test this.